modules/programs/hstr.nix

```
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let cfg = config.programs.hstr;
+in {
+  meta.maintainers = [ maintainers.Dines97 ];
+
+  options.programs.hstr = {
+    enable = mkEnableOption
+      "bash and zsh shell history suggest box - easily view, navigate, search and manage your command history";
```

Avoid long lines (also capitalization):

```suggestion
    enable = mkEnableOption ''
      Bash And Zsh shell history suggest box - easily view, navigate, search and
      manage your command history
    '';
```

---

Just copy pasted from the hstr github page to match this.